sa/sa_test.go

```diff
@@ -383,6 +383,10 @@ func TestAddCertificate(t *testing.T) {
 	test.Assert(t, certificateStatus.Status == core.OCSPStatusGood, "OCSP Status should be good")
 	test.Assert(t, certificateStatus.OCSPLastUpdated.IsZero(), "OCSPLastUpdated should be nil")
 
+	if CertStatusOptimizationsMigrated {
```

Comment:
If I set this to true in a unit test and the schema hasn't been migrated then the call to `sa.AddCertificate` will use the migrated model and fail. It doesn't seem possible to add a unit test that will exercise both models without the schema applied.